SupClient/src/setupHotkeys.ts

```diff
@@ -79,3 +85,8 @@ export default function() {
     hotkeyButton.title += ` (${hotkeyComplete})`;
   }
 }
+
+export function setupHelpCallback(callback: Function)

```

Comment:
When a function is this short, we usually write it on a single line.
Otherwise, the `{` should be at the end of the declaration line and not on the following line.